.gitignore

```diff
@@ -177,10 +177,12 @@ logs/
 *.dll
 dotnet_3/*/Properties/
 .vscode/
+.vs/
 *.launchSettings.json
 
 dotnet_3/em/embedded/rest/*/warning.html
 dotnet_3/cs/rest/Menu.API/Menu.API/packages.lock.json
+dotnet_3/*/logs/
```

Comment:
not sure about this one... what if a SUT has a code package called `logs`?  might be better to have explicetly an entry for each occurence of logs you want to ignore

Reply:
That's right. Thanks for pointing this out. I changed it.

Reply:
but `dotnet_3/*/logs/` is still here. should be removed.